apps/ca.c

```diff
@@ -722,8 +722,13 @@ int ca_main(int argc, char **argv)
 
     /*****************************************************************/
     if (req || gencrl) {
+        if (spkac_file != NULL) {
+            output_der = 1;
+            batch = 1;
+        }
         /* FIXME: Is it really always text? */
-        Sout = bio_open_default(outfile, 'w', FORMAT_TEXT);
+        Sout = bio_open_default(outfile, 'w',
```

Comment:
It seems like the FIXME can now be removed?

Reply:
Haha, yes!  How did I miss that?